subprojects/docs/src/samples/userguide/artifacts/maven/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,98 @@
+// tag::upload-file[]
+plugins {
+    // end::upload-file[]
+    java
+// tag::upload-file[]
+    maven
+}
+
+// tag::builder[]
+// tag::customize-pom[]
+// tag::multiple-poms[]
+tasks.getByName<Upload>("uploadArchives") {
+    repositories.withGroovyBuilder {
+        "mavenDeployer" {
+            "repository"("url" to "file://localhost/tmp/myRepo/")
+// end::upload-file[]
+// end::multiple-poms[]
+// end::builder[]
+            "pom" {
+                setProperty("version", "1.0Maven")
+                setProperty("artifactId", "myMavenName")
+            }
+// end::customize-pom[]
+// tag::builder[]
+            "pom" {
+                "project" {
+                    "licenses" {
+                        "license" {
+                            setProperty("name", "The Apache Software License, Version 2.0")
+                            setProperty("url", "http://www.apache.org/licenses/LICENSE-2.0.txt")
+                            setProperty("distribution", "repo")
+                        }
+                    }
+                }
+            }
+// end::builder[]
+// tag::multiple-poms[]
+            "addFilter"("api") {
+                delegate.withGroovyBuilder { setProperty("name", "api") }
```

Comment:
:x: Isn't that the same as just `setProperty("name", "api")`?

Reply:
Changed. It was wrong anyway, good catch.